hooks/use-progressive-transcripts.ts

```
@@ -0,0 +1,92 @@
+    } finally {
+      setIsLoadingTranscripts(false);
+    }
+  };
+
+  const retryLoading = () => {
+    if (!transcriptsLoaded && !isLoadingTranscripts) {
+      loadTranscripts();
+    }
+  };
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadTranscripts

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)